prometheus/prometheus.yml

```
@@ -0,0 +1,22 @@
+  - job_name: 'express'
+    static_configs:
+      - targets: ['app:3001']
+  - job_name: 'prometheus'
+    static_configs:
+      - targets: ['prometheus:9090']
+  - job_name: 'postgres'
+    static_configs:
+      - targets: ['postgres-exporter:9187']
+  - job_name: 'node'
+    static_configs:
+      - targets: ['node-exporter:9100'] 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)